File: Projekt_Python/zwierzeta/Wilk.py
from Zwierze import Zwierze

class Wilk (Zwierze): 
    def __init__(self, x, y,dane):
        if dane==None:
            self._x = x
            self._y = y
            self._xprev = x
            self._yprev = y
            self._symbol = 'W'
            self._sila = 9
            self._inicjatywa = 5
            self._zycie = 0
            self._wykonal_ruch=True
            self._numer=0
        else:
            self._x = dane[0]
            self._y = dane[1]
            self._xprev = dane[2]
            self._yprev = dane[3]
            self._inicjatywa = dane[4]
            self._symbol = 'W'
            self._sila = dane[5]
            self._wykonal_ruch = False
            self._zycie = dane[6]
            self._numer = dane[7]
        self._kolor= (105,105,105)

File: projekt_java/src/po/java/zwierzeta/Wilk.java
package po.java.zwierzeta;

import po.java.Zwierze;

import java.awt.*;

public class Wilk extends Zwierze {
    public Wilk(int x, int y)
    {
        this.x = x;
        this.y = y;
        this.xprev = x;
        this.yprev = y;
        this.symbol = 'W';
        this.sila = 9;
        this.inicjatywa = 5;
        this.zycie = 0;
        this.wykonal_ruch=true;
        this.numer=0;
        this.kolor= Color.DARK_GRAY;
    }
    public Wilk(int []dane)
    {
        this.x = dane[0];
        this.y = dane[1];
        this.xprev = dane[2];
        this.yprev = dane[3];
        this.inicjatywa=dane[4];
        this.symbol = 'W';
        this.sila = dane[5];
        this.wykonal_ruch = false;
        this.zycie = dane[6];
        this.numer=dane[7];
        this.kolor= Color.DARK_GRAY;
    }
}
